src/components/DayPickerNavigation.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { PropTypes } from 'react';
+import cx from 'classnames';
+
+import LeftArrow from '../svg/arrow-left.svg';
+import RightArrow from '../svg/arrow-right.svg';
+import ChevronUp from '../svg/chevron-up.svg';
+import ChevronDown from '../svg/chevron-down.svg';
+
+const propTypes = {
+  navPrev: PropTypes.node,
+  navNext: PropTypes.node,
+  isVertical: PropTypes.bool,
+
+  handlePrevMonthClick: PropTypes.func,
+  handleNextMonthClick: PropTypes.func,
+};
+const defaultProps = {
+  navPrev: null,
+  navNext: null,
+  isVertical: false,
+
+  handleNextMonthClick() {},
+  handlePrevMonthClick() {},
+};
+
+export default function DayPickerNavigation(props) {
+  const {
+    navPrev,
+    navNext,
+    isVertical,
+    handlePrevMonthClick,
+    handleNextMonthClick,
+  } = props;
+
+  let navPrevIcon = navPrev;
+  let navNextIcon = navNext;
+  let isDefaualtNavPrev = false;
```

Comment:
:astonished: I just mash buttons on a keyboard. It's a miracle this is my first typo.